.gitignore

```diff
@@ -22,6 +25,5 @@ notebooks/*/output/*json
 notebooks/*/output/*tsv
 notebooks/api-key.txt
 
-.coverage.*
-.coverage
-coverage.*
+tests/output/
+tests/input/*_conf.json
```

Comment:
@hrshdhgd I should have made a 'review comment' earlier when tagging you. I like being able to hit 'resolve' to clean the page up a bit.

Regarding `fhirjson_conf.json`, I located the source. It's from a (very helpful) unit test that @cmungall added in another PR recently. Basically, in order to keep the CLI clean, we're planning on having some dumpers, etc, pull their parameters from a config file rather than declaring them in `cli.py` in the normal `click` way. Right now, the only such dumper `output_type` that has such a config is `fhirjson`.

https://github.com/INCATools/ontology-access-kit/blob/22eebdf97472bdec44041ded5f4ebd8b5779355d/tests/test_cli.py#L477-L479

@cmungall It does look like this probably should be in the `.gitignore`, so I've added the following entry (though let me know if you actually do want to have these committed):
`tests/input/*_conf.json`